### PR TITLE
Setup hello microverse

### PR DESCRIPTION
**Pull request by allenkays**
In this update I:
- Removed section descriptions from README file
-  Added instructions for reproducing this project in local environment in the prerequisite and installation sections of the README file
-  Changed the tech stack to what I have used only.
-  Completed the linters setup adding the webhint and stylehint portions
-  Added package-lock.json and package.json files to ignore list 
- Added a license file
- Changed the branch name to a name of feature action.

#### Note: here is a link to the previous pull request that was closed on change of branch name.
[https://github.com/allenkays/Hello_Microverse/pull/1]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <title>Hello Microverse</title>
+    </head>
```
Comment (AbdusSattar-70) [inline index.html]: I acknowledge your dedication to implementing the previous reviewer's suggestion.🎉🎉 However, I would recommend utilizing the HTML5 meta tag to improve the overall quality. By including the following code snippet: your linter will successfully pass.
```suggestion
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
</head>
```

**Review by AbdusSattar-70 — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED
Hi @allenkays , 

Congratulations on successfully implementing the changes suggested by the previous reviewer! 👏 You have made significant progress in addressing some of the requested changes. However, there are still remaining items that need attention. As a result of the linter error, I regret to inform you that I am unable to proceed to the next project until all the required changes have been made. Please take the necessary steps to resolve the remaining issues so that we can move forward smoothly. Thank you for your understanding.

 ## Highlights
- professional readme file✔️
- Descriptive pr title and description✔️
- Correct use of GitHub workflow✔️
AlmostThereJumpingGIF

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (allenkays) [reply index.html]: Thanks for the great feedback. This is well noted and implemented now

**Review by victorokoroji — APPROVED** [review]
Hi @allenkays, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**